### PR TITLE
Eliminate Hungarian notation from variable names

### PR DESCRIPTION
**Pull request by clemens-tolboom**
I renamed some variable to make the code a little more readable. Hope that helps other too.

(the issue template disappeared; sorry about that)

### CODE REVIEW

**Comment by ermiyaeskandary** [comment]
Approved 

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/247/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Comment by Seple** [comment]
LGTM!

**Comment by ChadSki** [comment]
:+1: Hungarian notation is dead

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/247/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

File: bot.user.js [hunk 1]
```diff
@@ -754,13 +754,13 @@ var bot = window.bot = (function () {
...
             var aIndex = 0;
 
-            if (bot.currentFood && bot.currentFood.sz > bot.opt.foodAccelSz) {
+            if (bot.currentFood && bot.currentFood.sz > bot.opt.foodAccelSize) {
                 aIndex = bot.getAngleIndex(bot.currentFood.a);
 
                 if (
                     bot.collisionAngles[aIndex] && bot.collisionAngles[aIndex].distance >
                     bot.currentFood.distance + bot.snakeWidth * bot.opt.radiusMult
-                    && bot.currentFood.da < bot.opt.foodAccelDa) {
```
Comment (ChadSki) [inline bot.user.js:763]: What did `Da` even stand for? Distance-angle?

**Comment by ermiyaeskandary** [comment]
Hungarian ?

**Comment by ChadSki** [comment]
https://en.wikipedia.org/wiki/Hungarian_notation

> Hungarian notation is an identifier naming convention in computer programming, in which the name of a variable or function indicates its type or intended use.
> 
> `lAccountNum` : variable is a long integer ("l");
> `arru8NumberList` : variable is an array of unsigned 8-bit integers ("arru8");
> `szName` : variable is a zero-terminated string ("sz"); this was one of Simonyi's original suggested prefixes.
> `bReadLine(bPort,&arru8NumberList)` : function with a byte-value return code.

It boils down to "hard to read abbreviations of what type something is".

**Pull request merged**